cfme/scripting/appliance.py

```diff
@@ -37,7 +37,7 @@ def main():
 @click.argument('appliance-ip', default=None, required=False)
 @click.option('--cfme-only', is_flag=True, help='Upgrade cfme packages only')
 @click.option('--update-to', default='5.9.z', help='Supported versions 5.9.z,'
-              ' 5.10.z (.z means latest and default is 5.9.z)')
+              ' 5.10.z (.z means latest and default is 5.9.z)')  # leaving 59z support for upgrades
 def upgrade_appliance(appliance_ip, cfme_only, update_to):
```

Comment:
I think we can remove 5.7 check [here](https://github.com/ManageIQ/integration_tests/pull/8675/files#diff-e4474267b5ff1b1c04de4d64ee4afe24R55)(line no. 55)

Reply:
I'm leaving the older versions checks in this method, as its used for upgrade of appliances. I think we still want to support older versions where possible, but this likely needs more testing.